File: Selection-Sort.py
def SelectionSort(array):
    for i in range(len(array) - 1):
        minIndex = i
        for j in range(minIndex + 1, len(array)):
            if array[j] < array[minIndex]:
                minIndex = j
        array[i], array[minIndex] = array[minIndex], array[i]
    return array 

File: Selection-Sort.c
#include <stdio.h>

void SelectionSort(int array[], int n){
    int i, j, minindex, temp;
    for(i = 0; i < n; i++){
        minindex = i;
        for(j = i + 1; j < n; j++){
            if(array[minindex] > array[j]){
                minindex = j;
            }
        }
        temp = array[minindex];
        array[minindex] = array[i];
        array[i] = temp;
    }
}

int main(){
    int array[] = {4,2,3,1};
    int n = sizeof(array) / sizeof(array[0]);
    SelectionSort(array, n);
    for(int i = 0; i < n; i++){
        printf("%d, ", array[i]);
    }
}